README.md

```diff
@@ -4,12 +4,48 @@ Sunrise Java Shop Framework :sunrise:
 [![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-java.png?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-java) [![Stories in Ready](https://badge.waffle.io/commercetools/commercetools-sunrise-java.png?label=ready&title=Ready)](https://waffle.io/commercetools/commercetools-sunrise-java)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.commercetools.sunrise/commercetools-sunrise_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.commercetools.sunrise/commercetools-sunrise_2.11)
 
-The next generation shop template.
+The next generation shop framework. 
```

Comment:
That will be the next PR, this one is just focused on the installation part. I just fixed this because it was misleading.